File: app/src/main/java/com/example/twitterdemo/StarterApplication.java
package com.example.twitterdemo;

import android.app.Application;
import android.util.Log;

import com.parse.Parse;
import com.parse.ParseACL;
import com.parse.ParseException;
import com.parse.ParseObject;
import com.parse.ParseUser;
import com.parse.SaveCallback;

public class StarterApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        // Enable Local Datastore.
        Parse.enableLocalDatastore(this);

        // Add your initialization code here
        Parse.initialize(new Parse.Configuration.Builder(getApplicationContext())
                .applicationId("myappID")
                .clientKey("OOXtW6xkAxNg")
                .server("http://18.119.11.109/parse/")
                .build()
        );

        /*
        ParseObject object = new ParseObject("ExampleObject");
        object.put("myNumber", "123");
        object.put("myString", "Single");
        object.saveInBackground(new SaveCallback() {
            @Override
            public void done(ParseException e) {
                if (e == null){
                    Log.i("Parse Result", "Successful!");
                } else {
                    Log.i("Parse Result", "Failed" + e.toString());
                }
            }
        });
        */


        //ParseUser.enableAutomaticUser();
        ParseACL defaultACL = new ParseACL();
        defaultACL.setPublicReadAccess(true);
        defaultACL.setPublicWriteAccess(true);
        ParseACL.setDefaultACL(defaultACL, true);




    }
}

File: app/src/main/java/com/example/twitterdemo/StarterApplication.kt
package com.example.twitterdemo

import android.app.Application
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.parse.Parse
import com.parse.ParseACL
import com.parse.ParseException
import com.parse.ParseObject
import com.parse.SaveCallback

class StarterApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        // Enable Local Datastore.
        Parse.enableLocalDatastore(this)

        // Add your initialization code here
        Parse.initialize(Parse.Configuration.Builder(applicationContext)
                .applicationId("myappID")
                .clientKey("hCxsL9CG4krg")
                .server("http://3.70.112.251//parse/")
                .build()
        )


//        val `object` = ParseObject("ExampleObject")
//        `object`.put("myNumber", "123")
//        `object`.put("myString", "Single")
//        `object`.saveInBackground(SaveCallback() {
//            fun done(e: ParseException?) {
//                if (e == null) {
//                    Log.i("Parse Result", "Successful!")
//                } else {
//                    Log.i("Parse Result", "Failed $e")
//                }
//            }
//        })


        //ParseUser.enableAutomaticUser()
        val defaultACL = ParseACL()
        defaultACL.publicReadAccess = true
        defaultACL.publicWriteAccess = true
        ParseACL.setDefaultACL(defaultACL, true)

    }
}